runtime/executor/src/main/java/org/apache/nemo/runtime/executor/datatransfer/OperatorVertexOutputCollector.java

```diff
@@ -131,4 +132,33 @@ public void emitWatermark(final Watermark watermark) {
       }
     }
   }
+
+  @Override
+  public void emitLatencymark(final Latencymark latencymark) {
```

Comment:
Lot's of duplicate codes with emitWatermark. We need refactoring

Reply:
Does it means that we need to combine emitLatencymark and emitWatermark?

Reply:
I tried to merge it with a method called emitWatermark. However, until the methods of writer and transform are divided, merging only methods of OutputCollector looks not good. emitLatencymark and emitWatermark looks similar but eventually watermark and latencymark have to be handled differently. So I suggest remaining both of methods, emitLatencymark and emitWatermark.

Reply:
Or it's also welcome to think about a better structure together.